src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/HKDF.cs

```diff
@@ -81,14 +81,17 @@ private static void Extract(HashAlgorithmName hashAlgorithmName, int hashLength,
         /// </summary>
         /// <param name="hashAlgorithmName">The hash algorithm used for HMAC operations.</param>
         /// <param name="prk">The pseudorandom key of at least <see cref="HashLength"/> bytes (usually the output from Expand step).</param>
-        /// <param name="outputLength">The length of the output keying material.</param>
+        /// <param name="outputLength">The length of the output keying material that must be larger than 0.</param>
         /// <param name="info">The optional context and application specific information.</param>
         /// <returns>The output keying material.</returns>
         public static byte[] Expand(HashAlgorithmName hashAlgorithmName, byte[] prk, int outputLength, byte[]? info = null)
         {
             if (prk == null)
                 throw new ArgumentNullException(nameof(prk));
 
+            if (outputLength <= 0)
+                throw new ArgumentOutOfRangeException(nameof(outputLength), SR.Cryptography_OutputLength_OutOfRange);
```

Comment:
This should just reuse the existing `ArgumentOutOfRange_NeedPosNum` string

Reply:
Thanks! I missed that existing resource.